lib/complex_number.rb

```diff
@@ -0,0 +1,38 @@
+class ComplexNumber
+
+	include Math
```

Comment:
Hm, I am not sure this is a good idea to include Math module into your little class. Better use `require Math` at the beginning of the file. Read about difference between `require` and `include`